input/resources/codesystem-au-recorded-sex-or-gender-source-document-type.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<CodeSystem xmlns="http://hl7.org/fhir">
+    <id value="au-recorded-sex-or-gender-source-document-type" />
+    <meta>
+        <profile value="http://hl7.org/fhir/StructureDefinition/shareablecodesystem" />
+    </meta>
+    <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+        <valueInteger value="0"/>
+    </extension>
+    <url value="http://terminology.hl7.org.au/CodeSystem/au-recorded-sex-or-gender-source-document-type" />
```

Comment:
url is too long, doesn't follow convention of HL7 AU codesystems. None start with "au", most have abbreviations and do not separate each word. the shortest meaningful url is desirable.

This also means the id needs to change.

Reply:
Done.